cms/src/components/common/LCard.vue

```diff
@@ -53,7 +63,34 @@ function collapse() {
             </button>
         </div>
         <div v-show="!collapsed" data-test="collapsible-container">
+            <!-- Tab Navigation -->
+            <div v-if="tabs?.length" class="border-b">
```

Comment:
I think the tabs should rather be a separate component. The `LCard` doesn't need to be aware of any tabs, that can just be in the slot that gets passed into it. There is already a `LTabs` component in the codebase, but it's a bit half baked as it doesn't fully support switching tabs. It should be fairly easy to add that though, as it's already rendering slots. You can view the component in the [sandbox](http://localhost:4175/sandbox)

```html
                <LTabs :tabs="tabs" v-model:currentTab="selectedTab">
                    <template #tab-1>
                        Tab 1
                    </template>               
                    <template #tab-2>
                        Tab 2
                    </template>
                </LTabs>
```

-> this currently renders both tabs, but I think it will be pretty easy to add the correct functionality inside `LTabs`.
<img width="324" alt="image" src="https://github.com/user-attachments/assets/e6103da5-f190-4843-99a5-e19eebb7d58a">

Please also add tests for this component.